published/009-2016-March-28.md

```diff
@@ -0,0 +1,239 @@
+
+### Nginx metrics
+
+Our internal metrics dashboard didn't previously have HTTP request/response metrics from nginx's point of view, but only from IPFS's and [multireq](https://github.com/whyrusleeping/multireq)'s point of view. (Multireq is our v04x/v03x multiplexing proxy). Nginx itself provides finegrained metrics only through their commercial subscriptions. We're now using [mtail](https://github.com/google/mtail) to parse metrics from nginx access logs and expose them to [Prometheus](https://prometheus.io). @lgierth will also contribute the `nginx.mtail` program upstream with mtail.
+
+## Community
+
+### Upcoming talks
+
+On April 20th, IPFS will host a joint meetup with ConsenSys at MIT in Cambridge, Massachusetts. [Sign up here!](https://www.eventbrite.com/e/ipfs-ethereum-mit-meetup-tickets-24578852008)
+
+### First IPFS Meeting in NYC
+
+![Christian](009_christian.JPG)
+
+We had our first IPFS meetup in New York! It went fantastically; over 100 people came to the AWS Loft for a couple of hours to mingle, eat pizza, and hear the speakers:
+
+- @jbenet gave an introduction to IPFS and a talk about [IPLD](https://github.com/ipfs/specs/blob/master/merkledag/ipld.md).
+- [Christian Lundkvist](https://twitter.com/ChrisLundkvist) -- CTO of Consensys and author of many Ethereum applications (pictured above) -- gave an introduction to [uPort](https://medium.com/@ConsenSys/uport-the-wallet-is-the-new-browser-b133a83fe73#.fx13an2b0), an exciting new decentralized Identity system. uPort uses programs on the Ethereum Blockchain and content on IPFS to maintain identity records that are permanent, uncensorable, and self-controlled. The uPort attestation system allows third party authorities as well as peers to validate the user's information creating both a rich social fabric as well as powering solid KYC/AML systems in the financial sector.
+- Arkadiy gave an introduction to [Mediachain](https://blog.mine.nyc/introducing-mediachain-a696f8fd2035#.lh5fohuz1), a collaborative federated media metadata protocol that allows parties to make statements about creative works. The metadata statements are cryptographically signed by the contributor, timestamped in the Bitcoin blockchain, and stored in IPFS. The statements can then be looked up via perceptual search using an instance of the media itself.
+- @haadcode demo'd [Orbit](https://github.com/haadcode/orbit), a distributed p2p chat on IPFS.
+- [Chris Pacia](https://twitter.com/chrispacia) -- Lead backend developer for OpenBazaar -- spoke about the challenges of building a decentralized marketplace and the goals for OpenBazaar 2.0. The talk discussed how IPFS can help solve some key development challenges.
```

Comment:
it will be better to have this whole thing in its own post, that way we can give a large paragraph, pictures, and slides for all of the talks.